docs/faq.md

```diff
@@ -91,17 +91,17 @@
  
     !!!tip "Currently all bans have been for very obvious and intrusive actions, specifically:"
         - Developers using their unique Switch data to poke and reverse-engineer Nintendos Server responses
-        - People pirating games and using them online
+        - People pirating games and going online on emuNAND/SysCFW
```

Comment:
This change just makes the sentence more complicated while also being worded less clearly; installing a pirated game on emuMMC then going online in syscfw would be fine, but the way the sentence is worded here suggests installing a pirated game means you can now never go online ever or you'll be banned.